src/components/menu/menu.scss

```diff
@@ -38,6 +36,12 @@ ion-menu.show-menu {
   transform: translate3d(-9999px, 0, 0);
 
   contain: strict;
+
+  // scss-lint:disable PropertySpelling
+  @include multi-dir() {
+    left: 0;
+    right: auto;
+  }
```

Comment:
I did goof - the comment should be inside the mixin like - https://github.com/ionic-team/ionic/pull/11953